### PR TITLE
Adding setup & testing documentation and update deprecated attribute

### PR DESCRIPTION
**Pull request by puntope**
### Changes proposed in this Pull Request:

After finishing my env setup I added some more information regarding:

- `nvmrc` and `nvm use` in order to setup the correct node version
-  `npm Install && npm run dev` is a prerequisite to run the PHPUnit tests.
-  Updated from  `<p align:center>` to `<p style="text-align:center;">` at the end of Readme since this `align:center` attribute is obsolete. 

### Detailed test instructions:

1. Both section PHPUnit and Section Development make sense for you
2. You can clone the repo from scratch and run PHPUnit successfully following the guide
3. Made with 💜 section at the end of Readme is centred.

### Changelog entry

> Update - Readme - Setup and Testing information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,7 +152,7 @@ To remove the Docker container and images (this will **delete everything** in th
 
 -   [Usage Tracking](./src/Tracking/README.md)
 
-<p align="center">
+<p style="text-align: center">
```
Comment (tomalec) [inline README.md]: `style` seems to be removed by GitHub 
image
So I think we need to stick to deprecated `align`

**Review by tomalec — COMMENTED** [review]
Thanks for the update!

File: README.md (hunk 1 above)
Comment (puntope) [reply README.md]: Nice catch @tomalec!

It was working in my IDE built-in Markdown and confidently didn't check GitHub UI properly.  (my bad)

I rolled back to the previous version, let me know if the PR needs more improvements

**Review by tomalec — APPROVED** [review]
LGTM :)

**Comment by findingsimple** [comment]
Hey @puntope - just added you as the assignee

**Pull request merged**